src/Config/RuntimeConfigLoader.cs

```diff
@@ -107,7 +167,8 @@ public static JsonSerializerOptions GetSerializationOptions()
             PropertyNamingPolicy = new HyphenatedNamingPolicy(),
             ReadCommentHandling = JsonCommentHandling.Skip,
             WriteIndented = true,
-            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+            IncludeFields = true,
```

Comment:
What does this do?

Reply:
This allows fields to be serialized. We converted runtime config to have explicit fields